tools/packaging/macos/launcher/ROOT/Applications/PowerShell.app/Contents/Info.plist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleExecutable</key>
+	<string>PowerShell.sh</string>
+	<key>CFBundleGetInfoString</key>
+	<string>1.0</string>
+	<key>CFBundleIconFile</key>
+	<string>Powershell</string>
+	<key>CFBundleIdentifier</key>
+	<string>powershell</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
```

Comment:
should we be replacing any of these versions, with the one from the version/release tag?

Reply:
That value is related to the plist definition itself and should not change.

https://developer.apple.com/library/content/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102088